### ISSUE TITLE
Bug: geoproj, the image does not have sufficient information: skipping

### ISSUE THREAD

**Issue by githubhanjunjun** [DroneDB/DroneDB#198]
I execute the following command:
  ddb geoproj output/ images/DJI_0149.JPG 
error:
  	- 'VirtualXPath'	[XML Path Language - XPath]
  Cannot geoproject images/DJI_0149.JPG, the image does not have sufficient information: skipping
![image](https://user-images.githubusercontent.com/27795864/113106091-edc50980-9234-11eb-9624-e2f0f580040d.png)

**Comment by HeDo88TH** [comment]
Can you attach the image DJI_0149.JPG?

**Comment by githubhanjunjun** [comment]
> Can you attach the image DJI_0149.JPG?

![DJI_0149](https://user-images.githubusercontent.com/27795864/113111553-dee15580-923a-11eb-84ea-f4997f293a92.JPG)

**Comment by githubhanjunjun** [comment]
> Can you attach the image DJI_0149.JPG?

https://testcc.oss-cn-north-2.unicloudsrv.com/DJI_0149.JPG?AWSAccessKeyId=8eYv87JUpsxSiB6E&Expires=1617181813&Signature=WWHR%2FAmE7jr3KhmT5a8HMy4Uhq0%3D&x-oss-download=octet-stream

**Comment by pierotofy** [comment]
For some reason I can't download the image from the link. Could you upload it to https://dronedb.app instead?

**Comment by githubhanjunjun** [comment]
> For some reason I can't download the image from the link. Could you upload it to https://dronedb.app instead?

https://testhub1.dronedb.app/r/hanjunjun/91e877415fdf44c48339c3fc0cf1b672
![image](https://user-images.githubusercontent.com/27795864/113827110-815a8500-97b5-11eb-8fba-d08d693830c6.png)

**Comment by githubhanjunjun** [comment]
> For some reason I can't download the image from the link. Could you upload it to https://dronedb.app instead?

https://testhub1.dronedb.app/orgs/hanjunjun/ds/91e877415fdf44c48339c3fc0cf1b672/download/DJI_0149.JPG?inline=1

**Comment by pierotofy** [comment]
It looks like the camera's sensor size is not available in our lookup database (that's why geoproj can't project the image).

Was this captured with a Mavic 2 PRO? Or some other drone? (Which is it)?

Do you happen to know the camera sensor's dimensions (also known as CCD width)?

**Comment by githubhanjunjun** [comment]
> It looks like the camera's sensor size is not available in our lookup database (that's why geoproj can't project the image).
> 
> Was this captured with a Mavic 2 PRO? Or some other drone? (Which is it)?
> 
> Do you happen to know the camera sensor's dimensions (also known as CCD width)?

Looks like it's the Mavic 2 Pro.  I'm sorry. I'm not sure

**Comment by pierotofy** [comment]
I assume you didn't capture the image then? I can't find reliable reports that `dji fc2220` really is a mavic 2 pro. The only way to confirm would be to ask the person that captured the image.

**Comment by githubhanjunjun** [comment]
I can't trace the source of the photo, but is there any way to project the image onto the map?

**Comment by pierotofy** [comment]
Yes, but you need to know the camera sensor dimensions (which we don't..) and add it to the `sensor_data.sqlite`.

**Comment by githubhanjunjun** [comment]
> Yes, but you need to know the camera sensor dimensions (which we don't..) and add it to the `sensor_data.sqlite`.

Could you mention that you gave me some usable drone photos?

**Comment by pierotofy** [comment]
I'm not sure I understand; do you need sample photos? If so, check https://community.opendronemap.org/t/brighton-beach/56

**Comment by githubhanjunjun** [comment]
> I'm not sure I understand; do you need sample photos? If so, check https://community.opendronemap.org/t/brighton-beach/56

Could you please tell me how to realize the function in the picture below?
![image](https://user-images.githubusercontent.com/27795864/114976583-c9b22b00-9eb8-11eb-858c-03e39cf0f0ef.png)

**Comment by githubhanjunjun** [comment]
I don't know if the following camera parameters are    @pierotofy
![image](https://user-images.githubusercontent.com/27795864/114987142-67acf200-9ec7-11eb-9f27-7ec1a945e855.png)

**Comment by sbonaime** [comment]
> It looks like the camera's sensor size is not available in our lookup database (that's why geoproj can't project the image).
> 
> Was this captured with a Mavic 2 PRO? Or some other drone? (Which is it)?
> 
> Do you happen to know the camera sensor's dimensions (also known as CCD width)?

Hi,
I have the same problem. How do I add  my sensor data (DJI Zenmuse X7) to the lookup database ?

**Comment by HeDo88TH** [comment]
Feature request https://github.com/DroneDB/DroneDB/issues/247
